cookbooks/habitat/.kitchen.yml

```diff
@@ -0,0 +1,28 @@
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+
+verifier:
+  name: inspec
+
+platforms:
+  - name: centos-6.8
+  - name: centos-7.2
+  - name: debian-7.11
+  - name: debian-8.5
+  - name: fedora-24
```

Comment:
If we are vendoring the cookbook here rather than using a submodule does it still make sense to just leave it be? I don't personally have an opinion one way or the other.